.github/.OwlBot-hermetic.yaml

```diff
@@ -31,13 +31,13 @@ deep-copy-regex:
   dest: "/owl-bot-staging/v2/grpc-google-cloud-storage-v2/src"
 - source: "/google/storage/v2/.*-java/gapic-google-.*/src"
   dest: "/owl-bot-staging/v2/gapic-google-cloud-storage-v2/src"
-- source: "/google/storage/v2/google-cloud-storage-v2-java/gapic-google-cloud-storage-v2-java/src/main/java/com/google/storage/v2/gapic_metadata.json"
+- source: "/google/storage/v2/.*-java/gapic-google-.*/src/main/java/com/google/storage/v2/gapic_metadata.json"
```

Comment:
Do you know why `gapic_metadata.json` is not included by the previous entry?

Reply:
This entry means to move `gapic_metadata.json` from `main/java` to `main/resources`.

Looks like a library specific post processing.

Reply:
Yes, thank you. I oversaw this is a copy query into resources.